src/peergos/shared/user/fs/MimeTypes.java

```diff
@@ -43,7 +49,7 @@
     final static int[] WOFF = new int[]{'w','O','F','F'};
     final static int[] WOFF2 = new int[]{'w','O','F','2'};
 
-    final static int HEADER_BYTES_TO_IDENTIFY_MIME_TYPE = 28;
+    final static int HEADER_BYTES_TO_IDENTIFY_MIME_TYPE = 40;
```

Comment:
ideally we wouldn't increase this. I think we can get away with using an integer, and then having a lookup from integer to mimetype in code. cbor is a binary type after all.